tests/controller/deletecontent.js

```diff
@@ -233,12 +233,6 @@ describe( 'DataController', () => {
 				'<heading1>f[]</heading1><paragraph>x</paragraph>'
 			);
 
-			test(
```

Comment:
PS. Note that you haven't added this test in the "should leave a paragraph if the entire content was selected" block.